O365DataRetriever.ps1

```diff
@@ -1066,7 +1077,7 @@ $ConnectButton.Add_Click( {
                 IsDirSynced      = $Contact.IsDirSynced
                 RecipientTypeDetails = $Contact.RecipientTypeDetails
             }
-            $ContactsResults += New-Object PSObject -Property $ContactsProps
+            $script:ContactsResults += New-Object PSObject -Property $ContactsProps
         }
         $ContactsResults | Select-Object DisplayName, Company, Title, CountryOrRegion, PostalCode, IsDirSynced, RecipientTypeDetails
```

Comment:
@chwilfing: The contacts are not showing up in the tool, nor when you export to .csv 😢

Reply:
Found the issue. we reused the allcontacts variable which lead to an overlap in scoped variables. I fixed with not using a distinct allcontacts variable for the number and therefore no overwriting... should work now.

Reply:
Have you pushed your changes?

Reply:
Yes - see e1aaf46

Reply:
👍